samcli/commands/build/command.py

```diff
@@ -158,8 +158,19 @@ def do_cli(function_identifier,  # pylint: disable=too-many-locals
 
             click.secho("\nBuild Succeeded", fg="green")
 
-            msg = gen_success_msg(os.path.relpath(ctx.build_dir),
-                                  os.path.relpath(ctx.output_template_path),
+            # try to use relpath so the command is easier to understand, however,
+            # under Windows, when SAM and (build_dir or output_template_path) are
+            # on different drive, relpath() fails.
+            try:
+                build_dir = os.path.relpath(ctx.build_dir)
```

Comment:
nit: can you rename `build_dir` and `output_template_path` to be something more descriptive? I just don't want someone to use these values without knowing what it means.